github-dark.css

```diff
@@ -1956,7 +1956,7 @@
   .state-indicator.renamed, .discussion-topic .branch-status.status-pending,
   .compare-pr-placeholder, .compare-cutoff, .diff-cutoff, .flash.flash-warn,
   .flash-global.flash-warn, .markdown-body li.added.moved, .repo-private-label,
-  .gist-secret-label, .label-private, .stale-files-tab {
+  .gist-secret-label, .label-private, .RecentBranches, .stale-files-tab {
```

Comment:
The Element Inspector spewed that out (I was asked what class that was in #340 uploaded gif there but here it is again.

![classname](https://cloud.githubusercontent.com/assets/3521959/16853032/9217f220-4a02-11e6-82b8-7f98f62fe0f9.gif)

Im pretty sure that is right neither `recentbranches` or `recent-branches` works